doc/developer_guide/APIs.rst

```diff
@@ -1063,13 +1063,35 @@ TBD
 NEMO
 ====
 
+PSyIR Construction
```

Comment:
I couldn't find any mention to the process_nemo script and instructions about how it is currently executed in the documentation. Should there be a section here or in the nemo user's guide about it.

Maybe a more general question is what is the long term plan for nemo? Right now going to the example directory seems like a temporary/development thing. Is the plan to promote that script to a command line? or to integrate the functionality into the main psyclone command once it is clear who the new front-end/back-end will interface with LFRic? Maybe a note here in the developer guide about this will be helpful.

Reply:
Hmmm, good question. You are right that it is currently a temporary/development thing. It's also going to be very specific to the configuration of NEMO that is being built. @rupertford has suggested generalising the `process_nemo.py` script and promoting it to be something that we support and I think that makes sense. Really, the NEMO configuration-specific script ought to be the property of e.g. the NEMO build system (in the same way that LFRic has its own optimisation script in its repository). Until we achieve that, whatever we do feels a bit temporary. The same goes for the optimisation script itself (`kernels_trans.py`). I'll do some documenting...